installing/install_quick_roxctl.adoc

```diff
@@ -0,0 +1,58 @@
+[id="install_quick_roxctl"]
+= Installing quickly by using the `roxctl` CLI
+include::modules/common-attributes.adoc[]
+:context: install_quick_roxctl
```

Comment:
This context is still incorrectly `install_quick_roxctl`. Though I'm just noticing now that the file name  (and ID) also incorrectly use underscores.